project.clj

```diff
@@ -19,34 +19,8 @@
             [venantius/yagni "0.1.7"]]
   :cloverage {:ns-exclude-regex [#"api\.handler"]
```

Comment:
I think at this point, since `api.handler` is now testable, we should not exclude it from cloverage... and test it properly!

Reply:
Gonna see if it's possible.